src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");

```

Comment:
The message in the `doWork` method should indicate the type of machine more informatively. Consider adding 'Bulldozer' to the message to clarify which machine is working.